tests/test_invenio_mail_tasks.py

```diff
@@ -64,3 +70,46 @@ def test_send_message_stream(email_task_app):
                 'From: test2@test2.test2') != -1
             assert result_stream.getvalue().find('To: test2@test2.test2') != -1
             assert result_stream.getvalue().find('Subject: Test2') != -1
+
+
+def test_send_message_with_attachments(email_task_app):
+    """Test sending a message with attachments."""
+    with email_task_app.app_context():
+        filename = pkg_resources.resource_filename(
+            __name__, os.path.join('attachments', 'invenio.svg'))
+        content_type = 'image/svg+xml'
+        data = pkg_resources.resource_string(
+            __name__, os.path.join('attachments', 'invenio.svg'))
+
+        attachments = [Attachment(filename, content_type, data)]
+        msg = {
+            'subject': 'Test3',
+            'sender': 'test3@test3.test3',
+            'recipients': ['test3@test3.test3'],
+            'attachments': attachments
+        }
+
+        send_email(msg)
```

Comment:
Whoops. Stupid mistake. Good news is that it works with `delay` and `CELERY_ALWAYS_EAGER=True`. Let me test with everything set up.